Design Docs: System architecture and comparing to other tools (#265)
open-planning/roboplan: docs/source/design/comparison.rst

     - MoveIt Servo — a dedicated component, though in practice it tends to require substantial tuning to get good results
     - None
     - None
   * - Visualization
     - Viser web viewer with interactive gizmos; matplotlib plots
     - RViz plugin and Setup Assistant GUI
     - ``tesseract_qt`` / Tesseract Studio; RViz plugins
     - None (the OMPL.app GUI was removed in OMPL 2.0)

Which Tool Should You Use?
--------------------------

.. list-table::
   :header-rows: 1
   :class: comparison-table
   :widths: 12 44 44

   * - Tool
     - Choose it when...
     - Keep in mind...
   * - **RoboPlan**
     - You want a lean, code-first library for manipulator IK, planning, and trajectory generation that works in C++ or Python, with or without ROS.
     - Pre-1.0 with breaking changes and an intentionally small algorithm surface; no optimization-based planners, perception pipeline, or online execution layer (yet!).
   * - MoveIt 2
     - You are building a ROS 2 system and want a batteries-included stack: GUI setup, perception, execution, and swappable planner/IK plugins without recompiling.

>      - Requires ROS 2. Behavior is spread across many YAML/SRDF/launch files, and the abstract plugin interfaces can make debugging and customization harder. Open-source maintenance has slowed recently as development focus has shifted toward the commercial MoveIt Pro offering.

Reviewer: Maybe?

```suggestion
     - Requires ROS 2. Behavior is spread across many YAML/SRDF/launch files, and the abstract plugin interfaces can make debugging and customization harder. Open-source features and maintenance have slowed recently.
```

Author: Good call...
